doc/Language/community.pod6

```diff
@@ -275,20 +275,23 @@ If you prefer in-person meetings, these are warmly recommended!
 
 L<Camelia|https://raku.org/>, the multi-color butterfly with P 6 in her wings, is the symbol of this diverse and welcoming community.
 
-=head1 Raku Weekly
+=head1 Rakudo Weekly
 
-Elizabeth Mattijsen usually posts in L<the "Raku Weekly" blog|https://p6weekly.wordpress.com/>, a summary of Raku posts, tweets, comments and other interesting tidbits.
-If you want a single resource to know what is going on in the Perl community now, this is your best resource.
+Elizabeth Mattijsen usually posts in L<the "Rakudo Weekly" blog|https://https://rakudoweekly.blog/>, a summary of Raku posts, tweets, comments and other interesting tidbits.
+If you want a single resource to know what is going on in the Raku community now, this is your best resource.
 
+Historical articles (pre name change) can be found archived on
+L<the "Perl 6 Weekly" blog|https://p6weekly.wordpress.com/>.
 
-=head1 Raku Advent calendar
-
+=head1 Raku née Perl 6 Advent calendar
```

Comment:
My thought here was since I added a link to previous (Perl 6) advent calendars, it would be useful to mention this in the heading too, but I do see now that this isn't strictly necessary.